Tests/Functional/App/AppKernel.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the FOSElasticaBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Mango\Bundle\JsonApiBundle\Tests\Functional\App;
+
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpKernel\Kernel;
+
+/**
+ * Taken from https://github.com/FriendsOfSymfony/FOSElasticaBundle/blob/master/tests/Functional/app/AppKernel.php
```

Comment:
afaik original it's taken from https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/SecurityBundle/Tests/Functional/app/AppKernel.php#L23

Reply:
Yes, it is almost the same. Changed. Also, updated the file